src/scripts/Common/Base.cpp

```diff
@@ -679,6 +679,53 @@ EmoteDesc* MoonScriptCreatureAI::AddEmote(EventType pEventType, const char* pTex
     return NewEmote;
 }
 
+EmoteDesc* MoonScriptCreatureAI::AddEmote(EventType pEventType, uint32 scripttext)
+{
+    EmoteDesc* NewEmote = NULL;
```

Comment:
Use nullptr instead of NULL for types (Do not replace NULL with nullptr for old code! Just use it in new code)